dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/scalar/ByteToString.java

```diff
@@ -13,4 +28,33 @@
     public String eval(ByteString byteField) {
         return byteField.toStringUtf8();
     }
+
+    @Override
+    public TypeInference getTypeInference(DataTypeFactory typeFactory) {
+        TypeInference build = TypeInference.newBuilder()
+                .inputTypeStrategy(new InputTypeStrategy() {
+                    @Override
```

Comment:
Does it make sense to create a separate class for the `InputTypeStrategy` class? Would have made unit testing easier.

Reply:
yes